python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/management_security.schema.yml

```diff
@@ -115,6 +115,21 @@ keys:
               description: |
                 cipher_list syntax follows the openssl cipher strings format.
                 Colon (:) separated list of allowed ciphers as a string.
+                If using EOS version 4.32.0F or later, use the setting `ciphers` instead of `cipher_list`.
+            ciphers:
+              type: dict
+              description: This setting is applicable to EOS versions 4.32.0F and later.
```

Comment:
Both the descriptions of ciphers and ciphers_list say it can be used by 4.32.0F.

Reply:
Discussed